### ISSUE TITLE
Bug: Recipient doesn't receive prompt to accept the file transfer

### ISSUE THREAD

**Issue by anishmittal2020** [RobinLinus/snapdrop#168]
On [Snapdrop](https://snapdrop.net/), I tried to:

> Clicked the avatar of the recipient and selected the file I wanted to send. The file transfer should start once the recipient accepts the file.

But, I have tried both ways (sending from one to another) the recipient doesn't gets the prompt to accept it.

**System Configuration**: I am using the latest Chrome Browser on both Android 10 and Ubuntu 20.04.1 LTS x86_64.

**Comment by d4ni3l853** [comment]
I'm experiencing the same issue. Message sending doesn't work either. After a few seconds the light gray pulses stopped moving on both screens. Using Chrome 85.0.4183.101 on Android 9 and Chrome 85.0.4183.102 on Windows 10.

**Comment by najtin** [comment]
Just to be sure: Are you in the same home network? For example in the same wifi? Due to technicalities and security it wont work if the androids (i assume smartphones) are on a cellular network.

**Comment by d4ni3l853** [comment]
Yes, both devices are on the same wifi. I got it to work with Opera 71.0.3770.148 on the PC and the same Chrome for Android, so maybe it has something to do with Chrome for PC.

**Comment by anishmittal2020** [comment]
@najtin  Yes both devices are on the same network.

**Comment by penalte** [comment]
chrome in windows 10 is not working
